### ISSUE TITLE
[Quest] Gilneas - Take Back What's Ours + Preparations

### ISSUE THREAD

**Issue by Rushor** [WoWManiaUK/Blackwing-Lair#2552]
Preparations http://cata.cavernoftime.com/quest=24628 requires http://cata.cavernoftime.com/quest=24617

Take Back What's Ours http://cata.cavernoftime.com/quest=24646 requires At Our Doorstep http://cata.cavernoftime.com/quest=24627

Both prev quests must be added

**Comment by shadowfires** [comment]
Take Back What's Ours
https://www.wowhead.com/quest=24646/take-back-whats-ours

Dark Rangers are not at the Woods' End Cabin, which prevents the quest from being completed.

**Comment by Druhl** [comment]
Comment:

In my case I found the dark rangers present (about 20 total or so) but blowing the horn had no effect (did not summon worgen to assist). Tried twice. A non-interactive Tobias Mistmantle NPC was standing there already when I arrived in area but he took no actions. (Another was standing outside the entrance of Tal'doren.)

Was able to complete quest by running in, grabbing quest item, dying, and resurrecting out of range still with item.
